src/cursed_hr/cursed_hr.py

```diff
@@ -184,6 +184,18 @@ def __init__(
         self.configuration_index = 0
         self.use_prefix = use_prefix
 
+        self.priority_keys = {
+            "m": PenlogPriority.EMERGENCY,
```

Comment:
maybe capital `E` instead?

Reply:
Personally, I would prefer to leave it this way. Besides, the keyboard mapping is not related to this pull request. This part of the code is only refactored to avoid code duplication / incosistencies between the help message and the real keyboard mapping.